### ISSUE TITLE
How to use the code about "Encoder-Decoder"

### ISSUE THREAD

**Issue by wishinghao** [liren2515/DrapeNet#2]
Hello, I have some questions regarding the code in the "Encoder-Decoder" section. Is it used for training and extracting results? Could you provide a pre-trained model for fitting meshes or images?

**Comment by liren2515** [comment]
Hello, 'Encoder-Decoder' is used for training and extracting the latent codes (and reconstructed meshes if needed).

The pre-trained models for fitting are already included in the folder `checkpoints`.

**Comment by wishinghao** [comment]
When I run the "train-ids.pkl", I get "No such file or directory: 'train-ids.pkl'". How can I get this pkl file?

**Comment by wishinghao** [comment]
How to train the deformation field? I think the ∆x should be trained, but I can not find reference code.

**Comment by liren2515** [comment]
The training codes for $\Delta_x$ are not provided in this repo. Generally, we follow the physic-based loss from [SNUG](https://github.com/isantesteban/snug) to train it. The training details can be found in our paper.

**Comment by iyyovvv** [comment]
@wishinghao I have the same question.
When I run the "train-ids.pkl", I get "No such file or directory: 'train-ids.pkl'".

**Comment by LAW1223** [comment]
Thank you for your excellent work, I have some questions about the dataset. The file structure of CLOTH3D is every .obj is belongs to a fold. In this repo, should I put this .obj all together into one fold, according to the preprocess_udf.py

**Comment by fatbao55** [comment]
@wishinghao I also encountered "No such file or directory: 'train-ids.pkl'". Do you mind sharing how you obtained the `train-ids.pkl`?

**Comment by wishinghao** [comment]
> 

I have rewritten the dataloader, so I don't need the pkl file.

**Comment by wishinghao** [comment]
I have rewritten the dataloader, so I don't need the pkl file. @fatbao55

**Comment by liren2515** [comment]
> @wishinghao I also encountered "No such file or directory: 'train-ids.pkl'". Do you mind sharing how you obtained the `train-ids.pkl`?

The training files for Encoder-Decoder are provided [here](https://drive.google.com/file/d/1SIY9X1g_N-j7Vm47fowiiqTe5gIsCUEV/view?usp=sharing).
